### PR TITLE
Split the QGIS server tab in project properties

### LINKED ISSUE #7261: Split the QGIS server tab in project properties (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#46624
Author: @DelazJ
QGIS version: 3.24

**Split the QGIS server tab in project properties**

### PR Description:
into smaller service-oriented tabs (for you @gioman)
Screen cast (at the time of recording, I didn't include fixes in #46594 - Done later, to avoid ui conflict)

https://user-images.githubusercontent.com/7983394/147295189-d2ff718d-89ff-4f5c-9252-86e6b68fa807.mp4

There's some resizing issue at the opening (see services scrollbar), but once you switch tabs it disappears. I spent enough time on that and would let some more skilled people find what could be the origin.
The WMS tab still needs some reorganization but here also I'll let that to people that know the topic.
Supersedes #46594 Fixes #40507
Updated screenshot, integrating suggestions
image

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by PhilipAlbrecht**
Goal: Fixes #7261

Ticket(s): #

- [ ] Backport to LTR documentation is requested

### CODE REVIEW

**Comment by DelazJ** [comment]
@PhilipAlbrecht While addressing a new issue, you should start your branch from master and not from the previous one you used

**Comment by DelazJ** [comment]
I think we need more here:
- [ ] there's an update needed at https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#qgis-server-properties (where I would use your screenshot, maybe with the "Test configuration" menu also expanded)
- [ ] the screenshot in https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html#creatingwmsfromproject which you are updating here could display the WMS tab
- [ ] the WFS tab needs renaming in the text
- [ ] (optional) Question: does it make sense to display tabs for each subsection of https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html#creatingwmsfromproject? In which case the first screenshot would display the "service capabilities" + "Test configuration" menus expanded.

**Comment by PhilipAlbrecht** [comment]
> * [ ]  there's an update needed at https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#qgis-server-properties (where I would use your screenshot, maybe with the "Test configuration" menu also expanded)

- updated with expanded Test configuration

> * [ ]  the screenshot in https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html#creatingwmsfromproject which you are updating here could display the WMS tab

- i did a screenshot for all tabs now

> * [ ]  the WFS tab needs renaming in the text

- which part of the text do you mean where WFS tab needs remaining. couldn't find a part where it could be mentioned in this chapter. i added a wmts theme..did mean this? 


> * [ ]  (optional) Question: does it make sense to display tabs for each subsection of https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html#creatingwmsfromproject? In which case the first screenshot would display the "service capabilities" + "Test configuration" menus expanded.

- i did a screenshot for all tabs now. let me know what you think

File: docs/server_manual/getting_started.rst [hunk 1]
```diff
@@ -910,21 +918,55 @@ expressions) this will be listed inside the GetFeatureInfo output. If the
...
+   :align: center
+
+   Definitions in the WMTS tab
+
+If you enter a URL in the :guilabel:`Advertised URL` field of the
+:guilabel:`WMTS capabilities` section, QGIS Server will advertise this specific
+URL in the WMTS GetCapabilities response.
+
 WFS capabilities
 ----------------
```
Comment (DelazJ) [inline docs/server_manual/getting_started.rst:939]:
```suggestion
WFS/OAPIF capabilities
----------------------
```

File: docs/server_manual/getting_started.rst [hunk 2]
```diff
@@ -910,21 +918,55 @@ expressions) this will be listed inside the GetFeatureInfo output. If the
...
 
-In the :guilabel:`WFS capabilities` area you can select the layers you
+In the :guilabel:`WFS capabilities` tab, you can select the layers you
 want to publish as WFS, and specify if they will allow update, insert and
```
Comment (DelazJ) [inline docs/server_manual/getting_started.rst:942]:
```suggestion
In the :guilabel:`WFS/OAPIF capabilities` tab, you can select the layers you
want to publish as WFS or OAPIF, and specify if they will allow update, insert and
```

File: docs/server_manual/getting_started.rst [hunk 3]
```diff
@@ -910,21 +918,55 @@ expressions) this will be listed inside the GetFeatureInfo output. If the
...
 delete operations.
+
+.. _figure_wfs_definitions:
+
+.. figure:: img/ows_server_wfs.png
+   :align: center
+
+   Definitions in the WFS tab
```
Comment (DelazJ) [inline docs/server_manual/getting_started.rst:950]:
```suggestion
   Definitions in the WFS/OAPIF tab
```

File: docs/server_manual/getting_started.rst [hunk 4]
```diff
@@ -818,10 +821,7 @@ styles of the layers in QGIS and the project CRS, if not already defined.
 
    Definitions for a QGIS Server WMS/WFS/WCS project
 
-Then, go to the :guilabel:`QGIS Server` menu of the
-:menuselection:`Project --> Properties...` dialog and provide
-some information about the OWS in the fields under
-:guilabel:`Service Capabilities`.
+You have to :guilabel:`Enable Service Capabilities` first, if it is deactivated. 
 This will appear in the GetCapabilities response of the WMS, WFS or WCS.
 If you don't check |checkbox| :guilabel:`Service capabilities`,
```
Comment (DelazJ) [inline docs/server_manual/getting_started.rst:826]:
```suggestion
If you don't check |checkbox| :guilabel:`Enable Service capabilities`,
```

File: docs/server_manual/getting_started.rst [hunk 5]
```diff
@@ -810,6 +810,9 @@ Configure your project
 To provide a new QGIS Server WMS, WFS or WCS, you have to create a QGIS project
```
Comment (DelazJ) [inline docs/server_manual/getting_started.rst:810]:
```suggestion
To provide a new QGIS Server WMS, WFS, OAPIF or WCS, you have to create a QGIS project
```

**Review by DelazJ — COMMENTED** [review]
> which part of the text do you mean where WFS tab needs remaining. 

OAPIF is now part of the tab name

**Pull request merged**